tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Actions/Test_FluxCommunication.cpp

```diff
@@ -379,20 +403,19 @@ SPECTRE_TEST_CASE("Unit.DiscontinuousGalerkin.Actions.FluxCommunication",
   ActionTesting::set_phase(make_not_null(&runner),
                            metavariables::Phase::Testing);
 
-  runner.next_action<my_component>(self_id);
+  runner.template next_action<my_component>(self_id);
```

Comment:
[optional] there is now a non-member `ActionTesting::next_action` if you want to use that. Up to you :)